src/layouts/home-assistant-main.html

```diff
@@ -107,6 +110,22 @@
     }
   },
 
+  _hassChanged: function (hass) {
+    if (hass && !this._themeLoaded) {
+      this._themeLoaded = true;
+      var _this = this;
+      hass.callApi('get', 'themes').then(function (result) {
+        if (result.current_theme === 'default') return;
+        var theme = result.themes[result.current_theme];
+        Object.keys(theme).forEach(function (key) {
+          var value = theme[key];
+          _this.customStyle['--' + key] = value;
```

Comment:
Don't use customStyle. Pass the values into `updateStyles` instead: https://www.polymer-project.org/2.0/docs/upgrade#use-updatestyles-instead-of-customstyle